tools/Randovania Installer.aip

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<DOCUMENT Type="Advanced Installer" CreateVersion="19.7.1" version="21.1" Modules="professional" RootPath=".." Language="en" Id="{370A9BFE-2004-4093-B731-5DC292A8CF36}">
+<DOCUMENT Type="Advanced Installer" CreateVersion="19.7.1" version="22.0" Modules="professional" RootPath=".." Language="en" Id="{370A9BFE-2004-4093-B731-5DC292A8CF36}">
   <COMPONENT cid="caphyon.advinst.msicomp.ProjectOptionsComponent">
-    <ROW Name="HiddenItems" Value="AppXAppDetailsComponent;FixupComponent;AppXCapabilitiesComponent;AppXDependenciesComponent;AppXProductDetailsComponent;AppXVisualAssetsComponent;AppXAppDeclarationsComponent;AppXUriRulesComponent;MsiXDiffComponent;MsixManifestEditorComponent"/>
     <ROW Name="MigrationUninstallCode" Value="{0E1FAF99-6B2D-45B4-A2CF-64EB38A51D48}"/>
   </COMPONENT>
   <COMPONENT cid="caphyon.advinst.msicomp.MsiPropsComponent">
     <ROW Property="AI_APP_FILE" Value="[#randovania.exe]"/>
     <ROW Property="AI_BITMAP_DISPLAY_MODE" Value="0"/>
     <ROW Property="AI_FINDEXE_TITLE" Value="Select the installation package for [|ProductName]" ValueLocId="AI.Property.FindExeTitle"/>
+    <ROW Property="AI_PREDEF_LCONDS_PROPS" Value="AI_DETECTED_INTERNET_CONNECTION"/>
```

Comment:
Unrelated but i'd be curious to know what this property is supposed to do lol

Reply:
A detected internet connection being required to launch the installer? I did get a prompt mentioning that such a requirement would be enabled (since the dotnet installer is downloaded in runtime), but I think it was already enabled.

(Also `AI` is Advanced Installer. Took me a moment to realise)

Reply:
Oh, AI being advanced installer instead of artificial intelligence makes a lot more sense.